src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/wiremock/PrisonApiMockServer.kt

```diff
@@ -125,6 +141,43 @@ class PrisonApiMockServer : WireMockServer(8979) {
     )
   }
 
+  fun stubGetValidAdjudicationsById(testDataSet1: AdjudicationIntTestDataSet, testDataSet2: AdjudicationIntTestDataSet) {
```

Comment:
I could have made this more generic and reusable by passing a list of data sets, but IMO that is going too much in to the realms of writing another app.

Best to minimise the magic in tests.